axon-server-connector/src/main/java/org/axonframework/axonserver/connector/event/axon/EventProcessorInfoConfiguration.java

```diff
@@ -37,13 +37,10 @@
  */
 public class EventProcessorInfoConfiguration implements ModuleConfiguration {
 
-    private Configuration config;
-
-    private final Component<EventProcessingConfiguration> eventProcessingConfiguration;
-    private final Component<AxonServerConnectionManager> connectionManager;
-    private final Component<AxonServerConfiguration> axonServerConfiguration;
     private final Component<EventProcessorControlService> eventProcessorControlService;
 
+    private Configuration config;
```

Comment:
I assumed this class to be deprecated entirely, in favor of the `EventProcessorControlService`. Did I miss some new findings why we still need this `ModuleConfiguration`? :-)